jitter.json

```diff
@@ -1,115 +0,0 @@
-{
-  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
-  "data": {
-    "values": "<DVC_METRIC_DATA>"
-  },
-  "title": "<DVC_METRIC_TITLE>",
-  "width": 300,
-  "height": 300,
-  "transform": [
-    {"calculate": "random()", "as": "randomX"},
-    {"calculate": "random()", "as": "randomY"},
-    {"sample": 500}
```

Comment:
@dberenbaum I just remembered that we need this 🤦🏻. Remember [this](https://github.com/iterative/vscode-dvc-demo/pull/10#discussion_r1027508693). Is there a way to downsample using the dvc.yaml?

Reply:
🤦 Sorry, I should have checked that. I'll replace the plot with something that is already sampled to a reasonable amount of data.